scipy/stats/morestats.py

```diff
@@ -1816,6 +1838,13 @@ def rootfunc(ab, xj, N):
         logsf = distributions.gumbel_r.logsf(w)
         sig = array([25, 10, 5, 2.5, 1])
         critical = around(_Avals_gumbel / (1.0 + 0.2/sqrt(N)), 3)
+    elif dist == 'weibull_min':
+        c, loc, scale = distributions.weibull_min.fit(y, loc=0.0)
+        logcdf = distributions.weibull_min.logcdf(x=y, c=c, loc=loc, scale=scale)
+        logsf = distributions.weibull_min.logsf(x=y, c=c, loc=loc, scale=scale)
+        shape_for_critical = min(np.round(np.divide(1.0, c), 2) * 100, 50)
```

Comment:
wouldn't this give a key error unless you hit one of the keys by chance?

Reply:
c in this context is m in the article. 

it is the way to map it's values to the table ones. But you are right I forget to convert it to an integer value as in _Avals_weibull keys. will fix that

Reply:
isn't there some kind of interpolation needed, e.g. if `100*c = 27`?

Reply:
your are right, fixed that

Reply:
ok, that works. (not sure if one needs `np.divide`)